nginx/templates/cache-blacklist.conf.tmpl

```diff
@@ -1,5 +1,7 @@
 # inspired by https://gist.github.com/a-vasyliev/de8ffc6c6aa74cdeadfe
 
+set $c_uri $args;
```

Comment:
Good point, I don't know why I didn't put in in here in the first place^^

Reply:
👍

Reply:
I had to search the internet for the meaning of `$args` and found that it is esentialy the same as `$query_string` which I find more apropriate. Dunno what you guys think.

https://nginx.org/en/docs/http/ngx_http_core_module.html#var_args